pkg/csi/snapshot_restore_steps_test.go

```diff
@@ -42,13 +43,13 @@ func (s *CSITestSuite) TestValidateArgs(c *C) {
 						}, nil),
 					f.versionOps.EXPECT().GetCSISnapshotGroupVersion().Return(
 						&metav1.GroupVersionForDiscovery{
-							GroupVersion: alphaVersion,
+							GroupVersion: common.SnapshotAlphaVersion,
```

Comment:
Is there value in adding tests with GroupVersion set to SnapshotBetaVersion and SnapshotStableVersion ?

Reply:
In this case no, again the benefits of using interfaces. This test is testing validate args. another method validatesnapshotclass would need these tests.

Reply:
LGTM. Thanks for clarifying. Yea, I did notice that this is testing ValidateArgs , so the version should not matter. I decided to go ahead and post the question instead of assuming .